.gitignore

```diff
@@ -1,3 +1,5 @@
 venv
 .vscode
-.idea
\ No newline at end of file
+.idea
+services/__pycache__/
+templates/index.html
```

Comment:
This is not what I meant. I was talking about undoing the changes of the `index.html`